TestApps/Samples/Samples/TreeViewEvents.cs

```diff
@@ -100,6 +100,17 @@ class ViewStatus
 		TextNode selectionRow;
 		bool dragging;
 
+		//defines the default height size based in the current font
+		//if will work if you use same font size in every row,
+		//if not, you'll need check in every OnGetRequiredSize
+		double defaultHeight;
+
+		public ExpandableTextCellView () 
+		{
+			var defaultLayout = new TextLayout() { Text = "A" };
+			defaultHeight = defaultLayout.GetSize().Height;
```

Comment:
This won't work correctly. Because it doesn't take all possible glyphs into account. For example the height of `A` is smaller than the height of `Ag`. It's not possible to measure the final height at this point.
Better measure the first line, as proposed here: https://github.com/mono/xwt/pull/794/files#r172651546